ambari-server/src/main/resources/stacks/BIGTOP/3.2.0/properties/stack_packages.json

```diff
@@ -0,0 +1,655 @@
+          "current_dir": "{0}/current/ranger-usersync/conf",
+          "component": "ranger-usersync"
+        }
+          ],
+          "spark": [
+          {
+          "conf_dir": "/etc/spark/conf",
+          "current_dir": "{0}/current/spark-client/conf",
+          "component": "spark-client"
+        }
+          ],
+          "flink": [
+          {
+          "conf_dir": "/etc/flink/conf",
+          "current_dir": "{0}/current/flink-client/conf",
+          "component": "flink-client"
+        }
+          ],
+          "solr": [
+          {
+          "conf_dir": "/etc/solr/conf",
+          "current_dir": "{0}/current/solr-server/conf",
+          "component": "solr-server"
+        }
+          ],
+          "superset": [
+          {
+          "conf_dir": "/etc/superset/conf",
+          "current_dir": "{0}/current/superset/conf",
+          "component": "superset"
+        }
+          ],
+          "tez": [
+          {
+          "conf_dir": "/etc/tez/conf",
+          "current_dir": "{0}/current/tez-client/conf",
+          "component": "tez-client"
+        }
+          ],
+          "zeppelin": [
+          {
+          "conf_dir": "/etc/zeppelin/conf",
+          "current_dir": "{0}/current/zeppelin-server/conf",
+          "component": "zeppelin-server"
+        }
+          ],
+          "zookeeper": [
+          {
+          "conf_dir": "/etc/zookeeper/conf",
+          "current_dir": "{0}/current/zookeeper-client/conf",
+          "component": "zookeeper-client"
+        }
+          ]
+        },
+          "conf-select-patching": {
+          "HBASE": {
+          "packages": ["hbase"]
+        },
+          "HDFS": {
+          "packages": []
+        },
+          "HIVE": {
+          "packages": ["hive", "hive-hcatalog"]
+        },
+          "KAFKA": {
+          "packages": ["kafka"]
+        },
+          "MAPREDUCE2": {
+          "packages": []
+        },
+          "OOZIE": {
+          "packages": ["oozie"]
+        },
+          "RANGER": {
+          "packages": ["ranger-admin", "ranger-usersync", "ranger-tagsync"]
+        },
+          "RANGER_KMS": {
+          "packages": ["ranger-kms"]
+        },
+          "SPARK": {
+          "packages": ["spark"]
+        },
+          "FLINK": {
+          "packages": ["flink"]
+        },
+          "SOLR": {
+          "packages": ["solr"]
+        },
+          "SUPERSET": {
+          "packages": ["superset"]
+        },
+          "TEZ": {
+          "packages": ["tez"]
+        },
+          "YARN": {
+          "packages": []
+        },
+          "ZEPPELIN": {
+          "packages": ["zeppelin"]
+        },
+          "ZOOKEEPER": {
+          "packages": ["zookeeper"]
+        }
+        },
+          "upgrade-dependencies" : {
+          "ATLAS": ["STORM"],
+          "HIVE": ["TEZ", "MAPREDUCE2", "SQOOP"],
+          "TEZ": ["HIVE"],
+          "MAPREDUCE2": ["HIVE"],
+          "OOZIE": ["MAPREDUCE2"]
+        }
+        }
+        }
```

Comment:
`Oozie`, `Storm`, `Sqoop` and `Superset` can be deleted, also use correct indent here.